addons/payment_authorize/static/src/js/payment_form.js

```diff
@@ -21,11 +21,13 @@ odoo.define('payment_authorize.payment_form', require => {
          */
         _getInlineFormInputs: function (providerId) {
             if (this.authorizeInfo.payment_method_type === "credit_card") {
+                let form = document.getElementById(`o_authorize_${this.formType}_${providerId}`) 
```

Comment:
This is essentially the definition of `this.el`, corresponding to the element the widget mounted on. So you could leave the form ID as-is and find the inputs with `this.el.querySelector(...)` instead.

Reply:
Note that, to be semantically correct, the inputs' IDs should also be adapted. But I believe it'd be too pedantic for a fix like this knowing that the problematic case will no longer be supported in 17.0 anyway.